node-src/tasks/upload.test.ts

```diff
@@ -335,92 +341,6 @@ describe('uploadStorybook', () => {
     expect(ctx.uploadedFiles).toBe(2);
   });
 
-  it('batches calls to uploadBuild mutation', async () => {
```

Comment:
This test appeared twice in this file, as a result of a bad merge. So I removed one of them.